pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -29,12 +29,13 @@
 
 @Slf4j
 public class PhiManifestService {
-    private static final String CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_STABLE_ID = "CONSENT_ADDENDUM_PEDIATRIC";
-    private static final String SOMATIC_CONSENT_TUMOR_PEDIATRIC_QUESTION = "SOMATIC_CONSENT_TUMOR_PEDIATRIC";
-    private static final String SOMATIC_ASSENT_ADDENDUM_QUESTION = "SOMATIC_ASSENT_ADDENDUM";
-    private static final String CONSENT_ADDENDUM_ACTIVITY_STABLE_ID = "CONSENT_ADDENDUM";
-    private static final String LMS_QUESTION_SOMATIC_CONSENT_ADDENDUM_TUMOR = "SOMATIC_CONSENT_ADDENDUM_TUMOR";
-    private static final String OS2_QUESTION_SOMATIC_CONSENT_ADDENDUM_TUMOR = "SOMATIC_CONSENT_TUMOR";
+    public static final String CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE = "CONSENT_ADDENDUM_PEDIATRIC";
```

Comment:
A bit of cleanup here to make it more clear what is an activity code vs. stable id for questions.